### DISCUSSION TITLE
How to view notifications about academic integrity issues

### DISCUSSION THREAD

**Discussion by Alex-Lecturio** [SafeExamBrowser/seb-server#137] [Q&A]
Hi,
I was not able to find information about how to view/assess cases with academic integrity issues. 
Are notifications sent to exam administrators when there is a violation in the student's behavior? Where can admins review the logs?

Thank you!
Alex

**Comment by anhefti** [comment]
Hi Alex

No, there isn't the possibility to apply a mail-server to SEB Server yet. 
SEB Server has a monitoring view of an Exam within the SEB Server GUI (Browser GUI). There you see your configured indicators  for an exam for each SEB client connection. Incidents go on top of the monitoring view so one has a good overview of what happens. Also in the Exam Monitoring view you can open a specific SEB client connection and see the logs of that SEB Client within a list. You can lock/unlock SEB there and some other features. 

Please read the full documentation about SEB Server monitoring in [the SEB Server user guide](https://seb-server.readthedocs.io/en/latest/monitoring.html)
